### DISCUSSION TITLE
GÜVENLİK

### DISCUSSION THREAD

**Discussion by Maryo55** [community/community#30172] [Other Feature Feedback, Questions, & Ideas]
Selamın aleyküm ben bu uygulamayı yeni kullanmaya ve keşfetmeye başladım bunun sebebi hesabıma birinin bağlanması ve istediği her türlü içeriğe ulaşması ,konuşması bununla alakalı detayları öğrenmek ne işlem yapılmış bunları görmek kiminle neler konuşmuş bunları bilmek istiyorum lütfen yardım eder misiniz 

_Community Team Translation: 
Hi, I've just started to use and explore this application because someone can connect to my account and access all kinds of content they want, talk, learn the details about it, see what has been done, see what has been done, I want to know what they talked to, please help?_

**Comment by ghostinhershell** [comment]
Hey @Maryo55 👋 

Thanks for participating! Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Comment by ghostinhershell** [comment]
Hey @Maryo55,

Hi there! Following up on the last post, we went ahead and translated your question and the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
